history-of-quakers/modernized/14-chapter-12.adoc

```diff
@@ -1127,6 +1168,10 @@ though they had nothing to charge him with,
 but that he was come from Salem to Boston to visit his sick friend;
 and for this pretended crime the following warrant was framed:
 
+[.embedded-content-document.legal]
+--
+
+[.letter-heading]
 To the constables of Boston, Charlestown, Maiden, and Lynn.
```

Comment:
Is the letter head okay here? Like I said, nothing seems right that I did!!

Reply:
yeah, another sort of weird one, but I agree, let's keep it.